Library/Formula/haxe.rb

```diff
@@ -26,7 +26,7 @@ class Haxe < Formula
   def install
     # Build requires targets to be built in specific order
     ENV.deparallelize
-    system "make", "OCAMLOPT=ocamlopt.opt"
+    system "make", "OCAMLOPT=ocamlopt.opt", ("ADD_REVISION=1" if build.head?)
```

Comment:
I don’t think this will work if `build.head?` is false because the last argument will then be `nil`:

```
>> system "ls", ("foo" if false)
TypeError: no implicit conversion of nil into String
	from (irb):1:in `system'
	from (irb):1
	from /usr/local/Library/brew.rb:124:in `<main>'
```